widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/base/HeaderRemoteConfig.kt

```diff
@@ -16,8 +16,6 @@ internal data class HeaderRemoteConfig(
     @SerializedName("closeButton")
     val closeButtonRemoteConfig: ButtonRemoteConfig?,
 
-//  "endScreenSharingButton" parameter is missing because it is located in notification(in IOS it is in AppBar)
```

Comment:
I suggest adding TODO here to add `endScreenSharingButton` param and its implementation, as that button is added to the header on your previous task, and we need to make it configurable